src/main/java/com/microsoft/graph/tasks/LargeFileUploadResponseHandler.java

```diff
@@ -76,7 +76,7 @@ public <ResponseType> LargeFileUploadResponse<UploadType> generateResult(
 			@Nonnull final IHttpRequest request,
 			@Nonnull final ResponseType response,
 			@Nonnull final ISerializer serializer,
-			@Nonnull final ILogger logger) throws Exception {
+			@Nonnull final ILogger logger) throws IOException, GraphServiceException {
```

Comment:
same comment about the binary breaking change. This one however is not going away on the next version. My suggestion is to rollback that change for the current version, add a suppression, and add an issue to clean up all the suppressions for exceptions for vnext.